xapian-core/queryparser/mathtermgenerator_internal.cc

```diff
@@ -0,0 +1,435 @@
+/** @file mathtermgenerator_internal.cc
+ * @brief MathTermGenerator class internals
+ */
+/* Copyright (C) 2018 Guruprasad Hegde
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301 USA
+ */
+
+#include <config.h>
+
+#include "mathtermgenerator_internal.h"
+
+/* #define DEBUGGING */
```

Comment:
Probably this shouldn't be checkedin

Reply:
This macro is useful for now. I don't know if there is a simple way to avoid part of the file from Git. Please suggest if you have any idea.